packages/cadl-ranch-specs/http/azure/resource-manager/operation-templates/available-operations.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.OperationTemplates;
+
+interface AvailableOperations {
+  @scenario
+  @scenarioDoc("""
+    Resource GET operation.
+    Expected path: /providers/Azure.ResourceManager.OperationTemplates/operations
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected response body:
+    ```json
+    {
+      "value": [{
+        "name": "Microsoft.Compute/virtualMachines/write",
+        "isDataAction": false,
+        "display": {
+          "provider": "Microsoft Compute",
+          "resource": "Virtual Machines",
+          "operation": "Create or Update Virtual Machine.",
+          "description": "Add or modify virtual machines.",
+        },
+        "origin": "user,system",
+        "actionType": "Internal",
+      }]
+    }
+    ```
+    """)
+  list is Azure.ResourceManager.Operations.list;
```

Comment:
Why is it deviating from https://github.com/Azure/cadl-ranch/pull/713/files#diff-777b6530b4d45deee68abaf86ad3c2e70640509d04bbb6c76fc6cee244cbf680?

Reply:
Fixed in the new version.